### PR TITLE
Update facts.txt

### PR DESCRIPTION
**Pull request by aseel-mhd**
Fixed previous conflicts

### CODE REVIEW

**Comment by rahuldecoded** [comment]
Why are you moving things here and there?

**Comment by aseel-mhd** [comment]
I sorted it alphabetically

File: facts.txt [hunk 1]
```diff
@@ -31,50 +27,59 @@
...
+	"There are 66 calories in 1 slice of White Bread. Calorie breakdown: 11% fat, 78% carbs, 11% protein."
+     },
+    "Wine": {
+        "Since red wine is fermented together with its skin, it has more antioxidants than white wine which is processed without its skin."
+        "Wine names usually indicate the location or the grape varieties used."
+        "Wine dilates the small blood vessels and helps to prevent angina and clotting."
+        "The average age of a French oak tree harvested for use in creating wine barrels is 170 years."
+        "10,000 varieties of wine grapes exist worldwide."
+    }
 }
```
Comment (rahuldecoded) [inline facts.txt]: Comma is missing

**Comment by rahuldecoded** [comment]
Would you like to contribute more in this project?

**Pull request merged**